packages/salesforce-adapter/src/filters/custom_object_split.ts

```diff
@@ -17,31 +17,44 @@ import _ from 'lodash'
 import { Element, isObjectType, ObjectType } from '@salto-io/adapter-api'
 import { pathNaclCase } from '@salto-io/adapter-utils'
 import { collections } from '@salto-io/lowerdash'
-import { isCustom, isCustomObject } from '../transformers/transformer'
-import { FilterWith } from '../filter'
+import { isCustom, isCustomObject, apiName } from '../transformers/transformer'
+import { FilterWith, FilterCreator } from '../filter'
 import { getObjectDirectoryPath } from './custom_objects'
+import { OBJECT_FIELDS_PATH } from '../constants'
 
 const { awu } = collections.asynciterable
 
 export const annotationsFileName = (objectName: string): string => `${pathNaclCase(objectName)}Annotations`
 export const standardFieldsFileName = (objectName: string): string => `${pathNaclCase(objectName)}StandardFields`
 export const customFieldsFileName = (objectName: string): string => `${pathNaclCase(objectName)}CustomFields`
 
-const customObjectToSplitElements = async (customObject: ObjectType): Promise<ObjectType[]> => {
-  const annotationsObject = new ObjectType({
-    elemID: customObject.elemID,
-    annotationRefsOrTypes: customObject.annotationRefTypes,
-    annotations: customObject.annotations,
-    path: [
-      ...await getObjectDirectoryPath(customObject),
-      annotationsFileName(customObject.elemID.name),
-    ],
-  })
+const perFieldFileName = (fieldName: string): string => `${pathNaclCase(fieldName)}`
```

Comment:
not sure this needs to be a function since it just calls `pathNaclCase`...
```suggestion
const perFieldFileName = (fieldName: string): string => pathNaclCase(fieldName)
```